tests/cases/fourslash/codeFixPromiseToAsync.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+////function [|f|]():Promise<any> {
+////    return fetch('http://yahoo.com').then(result => { console.log(result); });
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f() {

```

Comment:
By removing the return type annotation, your result now has a return type of `Promise<void>` rather than `Promise<any>` as was originally written.